packages/logging-bunyan/README.md

```diff
@@ -66,7 +66,29 @@ var loggingBunyan = require('@google-cloud/logging-bunyan')({
 // ...you're good to go!
 ```
 
+## Formatting Request Logs
+
+To format your request logs you can provide a `httpRequest` property on the bunyan metadata you provide along with the log message. We will format this as the [`HttpRequest`][http-request-message] message and Stackdriver logging will show this as a request log. Example:
```

Comment:
Instead of `we will format this`, maybe we should say, `we will treat this as` or `we will consider this`... the former might suggest we are going to take potentially invalid input and "correct" it.